plugins/main/bridge/bridge_test.go

```diff
@@ -906,4 +909,36 @@ var _ = Describe("bridge Operations", func() {
 			Expect(link.Attrs().HardwareAddr).To(Equal(origMac))
 		}
 	})
+
+	It("checks ip release in case of error", func() {
+		err := originalNS.Do(func(ns.NetNS) error {
+			tc := testCase{
+				cniVersion: "0.3.1",
+				subnet:     "10.1.2.0/24",
+			}
+
+			_, _, err := setupBridge(tc.netConf())
+			Expect(err).NotTo(HaveOccurred())
+
+			args := tc.createCmdArgs(originalNS)
+
+			// get number of allocated IPs before asking for a new one
+			files, _ := ioutil.ReadDir(ipamDatadir)
+			before := len(files)
+
+			debugPostIPAMError = fmt.Errorf("debugPostIPAMError")
+			_, _, err = testutils.CmdAddWithResult(originalNS.Path(), IFNAME, args.StdinData, func() error {
```

Comment:
`CmdAddWithResult` was renamed to `CmdAdd` in 731298003c55418f15a73044757a5e36985fae0f. It now has a slightly different signature.

Reply:
I rebased to master and changed this function by `CmdAddWithArguments`
Thanks!